halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/google/GoogleProviderUtils.java

```diff
@@ -50,12 +55,23 @@
 
 import static com.netflix.spinnaker.halyard.core.problem.v1.Problem.Severity.FATAL;
 
+@Slf4j
 class GoogleProviderUtils {
   // Map from service -> the port & job managing the connection.
   private static ConcurrentHashMap<String, Proxy> proxyMap = new ConcurrentHashMap<>();
   public static String getNetworkName() {
     return "spinnaker-hal";
   }
+  public static String getSshKeyFile() {
+    return Paths.get(System.getProperty("user.home"), ".ssh/google_compute_halyard").toString();
```

Comment:
Yup could be, I don't see the value (yet) in doing that so I'm skipping it for now.